.prettierignore

```diff
@@ -2,5 +2,6 @@
 src/lib/
 !/src
 !/react
+!/playwright
```

Comment:
Is there any reason to exclude `/playwright`?

Reply:
I tried to prettier test without this line.
But prettier was not applied.
I will check one more time.

Reply:
@yomybaby 
```!/playwright``` must exclude to apply prettier.
Because first line is ```/*```